extensions/msal-node-extensions/test/broker/NativeBrokerPlugin.spec.ts

```diff
@@ -1030,6 +1161,7 @@ if (process.platform === "win32") {
                         const result: AuthResult = {
                             idToken: "",
                             accessToken: "",
+                            authorizationHeader: "",
```

Comment:
Tons of duplicate code in this PR, can we do anything to eliminate the duplication?